src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -96,13 +98,11 @@ Learn more about CodeStream in our [short overview](https://newrelic.com/codestr
 </Collapser>
 </CollapserGroup>
 
-In addition to a suite of observability options, New Relic gives you the option to set up and report on custom data, like events and attributes to capture any data that you may want to see. You can setup these data points to be whatever you need them to be to match the needs of your organization. Check out our docs on [custom events and attributes](/docs/data-apis/custom-data/custom-events/report-custom-event-data/) to learn more.
-
 ## Pricing you want, functionality you need [#pricing]
 
-Having the capabilities you need in a data monitoring solution is important, but cost often proves to be just as pivotal a factor, if not more so. New Relic offers flexible, transparant pricing focused around your knees. We believe organizations of any size should have the access to the features they need to monitor data, so we base our pricing around the amount of data and the number of users, not availability of features. 
+Having the capabilities you need in a data monitoring solution is important, but cost often proves to be just as pivotal a factor. New Relic offers flexible pricing focused around your knees. We believe organizations of any size should have the access to the features they need, so we base our pricing around the amount of data and the number of users, not the number of features used. 
```

Comment:
"New Relic offers flexible pricing focused around your knees." ok, I laughed, because this is just really funny :) but you probably want to fix it.

I love the header of this section. I want this paragraph to have the same brief, punchiness.

How about something more directed at the reader? "It doesn't matter if a software service has all the capabilities you need, if it's out of your price range. We've worked hard to come up with flexible pricing that's focused around what you use, instead of the number of features you have access to." or something